templates/nft-template/apps/launchpad/components.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
+  "style": "default",
+  "rsc": true,
+  "tsx": true,
+  "tailwind": {
+    "config": "tailwind.config.js",
+    "css": "./app/globals.css",
+    "baseColor": "slate",
```

Comment:
NIT: Any strong feels against using the `zinc` theme instead? I think the more neutral monochromatic colors might be ideal for a template project. It's not a big deal, but I don't really love the strong blue tint of the `slate` theme in dark mode.

Reply:
sure! i was just using default shadcn configuration - we can adjust it how we want 😄